pyvista/core/filters.py

```diff
@@ -1912,6 +1912,9 @@ def extract_surface(dataset, pass_pointid=True, pass_cellid=True, inplace=False)
             surf_filter.PassThroughPointIdsOn()
         surf_filter.Update()
 
+        # need to add
+        # surf_filter.SetNonlinearSubdivisionLevel(subdivision)
```

Comment:
What's this?

Reply:
It's a note for a feature that I need to add but don't want to make another issue for.  For quadratic elements, extracting the surface doesn't yield you the linear representation but gives you a curved surface.  Makes for really smooth plots with high subdivision values.